src/utils/dispute-utils.js

```diff
@@ -306,6 +334,15 @@ export function getAdjudicationPhase(dispute, round, now, courtConfig) {
   }
 }
 
+/**
+ * @dev Terminology here will be:
+ *            Last round => dispute's last round
+ *            Final round => max possible round for a dispute (when the max appeals for a given dispute is reached)
```

Comment:
Nitpick: indentation here seems a bit weird.

I always have a hard time choosing how to indent these multi-line JSDoc strings, but usually 2 spaces past the start of the previous line looks OK (it is currently 6).